workshop.pl

```diff
@@ -197,7 +197,7 @@ sub run_command {
     my ($command) = @_;
 
     $command .= " 2>&1";
-    my $command_output = `$command`;
+    my $command_output = `. /etc/profile; $command`;
```

Comment:
Hmm, aren't all the changes in this file in this PR redundant to the other PR I just looked at?  Not sure what kind of conflict this might cause...